pkgs/by-name/sy/systemd-lsp/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "systemd-lsp";
+  version = "2025-06-25";
+
+  src = fetchFromGitHub {
+    owner = "JFryy";
+    repo = "systemd-lsp";
+    rev = "9834de8b338796755e5d3bd0dede5075c11662b9";
+    sha256 = "sha256-xKVmpjcKAvoyBUd7W5sl72YiPtj/7U315IkbCz0njHw=";
+  };
+
+  cargoHash = "sha256-bYksgHTXomeEJuSk800+/PYXzMvrixSjfPnoqxStWAA=";
+
+  meta = {
+    description = "Language server implementation for systemd unit files made in Rust.";
```

```suggestion
    description = "Language server implementation for systemd unit files made in Rust";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description